src/index.test.ts

```diff
@@ -26,6 +26,7 @@ describe('the default export', function () {
     const values = ['a', 'b', 'c'];
     expect(values).toContain(fake.sample(values));
     expect(fake.digit()).toEqual(expect.any(Number));
+    expect(fake.state()).toEqual(expect.any(String));
```

Comment:
There's a comment at the top of this file that says this file shouldn't really exist! We should add be adding new/updated tests to their own method-specific files.